packages/rspack/tests/runtimeCases/runtime/split-css-chunk-with-content-hash/webpack.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
 	output: {
-		chunkFilename: "[id].[contenthash][ext]"
+		chunkFilename: "[id].[contenthash].js"
```

Comment:
@hyf0 @hardfist The `webpack` is not support `[ext]` at `chunkFilename`, but rspack previous release is supported, Look like it is a break change.